### PR TITLE
[lager] update to latest

### PR DESCRIPTION
**Pull request by altschuler**
**Describe the pull request**

- #### What does your PR fix?  
  Updates lager to latest master

- #### Which triplets are supported/not supported? Have you updated the [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt)?  
  all, yes

- #### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?  
  yes (I'm not the original contributor of the lager package, so did not create the initial port)

- #### If you have added/updated a port: Have you run `./vcpkg x-add-version --all` and committed the result?  
  yes (or rather I create the version entry manually because I didn't know about `x-add-version` then)

### CODE REVIEW

File: ports/lager/vcpkg.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "lager",
-  "version-date": "2021-06-17",
-  "port-version": 1,
+  "version-date": "2022-02-21",
+  "port-version": 2,
```
Comment (JonLiu1993) [inline ports/lager/vcpkg.json:4]:
```suggestion
```

**Comment by JonLiu1993** [comment]
@altschuler ,Thanks for your pr, please run command "./vcpkg x-add-version lager --overwrite-version" and commit again

**Comment by altschuler** [comment]
That seemed to reset the port-version to `0` and added another version, I think I've fixed it up now 😅

**Comment by JonLiu1993** [comment]
@altschuler ,it's all good now

**Review by JonLiu1993 — APPROVED** [review]

**Comment by altschuler** [comment]
Cool, thanks! I'm not sure I understand the port-version, it's now `0` again, so it went from `0` -> `1` -> `0` in the `versions` list?

**Comment by JonLiu1993** [comment]
> Cool, thanks! I'm not sure I understand the port-version, it's now `0` again, so it went from `0` -> `1` -> `0` in the `versions` list?

You can refer to [this ](https://github.com/microsoft/vcpkg/blob/876e67c26e42c0c7d2daa41c6871af117ae6bec5/docs/maintainers/control-files.md#version)

**Comment by altschuler** [comment]
> > Cool, thanks! I'm not sure I understand the port-version, it's now `0` again, so it went from `0` -> `1` -> `0` in the `versions` list?
> 
> You can refer to [this ](https://github.com/microsoft/vcpkg/blob/876e67c26e42c0c7d2daa41c6871af117ae6bec5/docs/maintainers/control-files.md#version)

Perfect, thank you! This was an extremely fast and efficient review :)

**Pull request merged**